package.json

```diff
@@ -23,13 +23,14 @@
   "license": "Apache-2.0",
   "dependencies": {
     "basic-auth-parser": "0.0.2",
-    "bluebird": "^3.5.2"
+    "bluebird": "^3.5.3",
+    "pjson": "^1.0.9"
   },
   "devDependencies": {
     "@types/bluebird": "^3.5.24",
-    "@types/chai": "^4.1.6",
+    "@types/chai": "^4.1.7",
     "@types/mocha": "^5.2.5",
-    "@types/node": "^8.10.34",
+    "@types/node": "^10.12.9",
```

Comment:
Do you need this bump?  Arguably it should wait for a major version bump in order to help ensure we don't break compatibility with node 8 in a patch/minor